### ISSUE TITLE
Youtube Post Being "locked as private"

### ISSUE THREAD

**Issue by MinicMedia** [inovector/mixpost#75]
### Detailed description

Any time i post a video through the app my video finishes the upload and it makes it to youtube site. But they are locked as private. I also get sent a email from youtube every time. Saying that it violates their spam, deceptive practices and scams policy. There is also no way to appeal it. Is there a step i may have missed?

**Comment by lao9s** [comment]
@MinicMedia Unfortunately, we cannot help you because the YouTube video processor decides how to publish.

**Comment by MelonSmasher** [comment]
I know this is closed, but I'm running in to the same problem, also just payed for this and I'm not able to use it. I wanted to note that when I upload the same exact videos with the same exact descriptions and titles manually through the web UI the video posts fine. Not sure if anyone has some insight?

**Comment by MinicMedia** [comment]
Im glad that someone else ran into this same issue. No matter what i did i couldn't figure it out. All the videos were getting blocked. One thing that worked was to make sure the size of your video is 9x16 and exported as 1080x1920. I had like a 10% success rate with those settings. Eventually i just gave up with youtube and decided to post those same videos manually.

**Comment by MelonSmasher** [comment]
@MinicMedia so you experienced the same behavior where the same video would work manually? The video is 9:16 @ 1080x1920

**Comment by MinicMedia** [comment]
Yes, is the video h.264? HVEC and prores did not work for me. I will retry posting a video right now and re-confirm. 

Just did the test and looks like it did post to my YouTube channel!

Steps i did:
- Post only to one platform 
- Video that i used is 16x9 and 61.9 MB
- H.264
I made sure that i was only posting to youtube.

Would you be able to share the file that you are trying to post?

**Comment by MelonSmasher** [comment]
> is the video h.264
yes.

I did a bit more troubleshooting and I believe the problem was where I was hosting the Mixpost instance or rather YouTube did not like that Mixpost was uploading content from my Digital Ocean instance. I moved everything to my local network where I was able to manually upload content and then after that post did not get locked.

Just something else for people to keep in mind if they run in to this.
